components/AppBar.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
+import { StyleSheet, View } from 'react-native'
+import { Appbar, Text, Menu } from 'react-native-paper'
+
+import { useAuthenticationContext } from '../context/authentication'
+import theme from '../defaultTheme'
+
+const UI_STRINGS = {
+  logout: 'Logout',
+}
+
+export default function AppBar() {
+  const [isMenuActive, setMenuActive] = useState(false)
+  const { handleLogout } = useAuthenticationContext()
+  return (
+    <Appbar>
+      <Appbar.Content title={<Text style={styles.title}>Optic</Text>} />
```

Comment:
let's be consistent with hardcoded strings. either hardcoded or in a constant lookup